### ISSUE TITLE
Syscall 13-16 specifications

### ISSUE THREAD

**Issue by tgrossb** [sbustars/STARS#5]
This is not an error but rather a request for more available information on the specifications of syscalls 13-16 that are responsible for file access.

For example, looking at syscall 13 (open file), the flags are not currently documented and require checking the syscall interpreter functions to identify that read and write flags are the inverse of what they are in the most accessible [MARS documentation](https://courses.missouristate.edu/kenvollmar/mars/help/syscallhelp.html).  Similarly, error handling such as attempting to open non-existing or non-accessible files results in a program terminating error while documentation of other MIPS assemblers would suggest that an error would quietly be indicated through a negative file descriptor, a negative result in return registers, etc.

While these can be found through trial and error or checking code, I think it would be useful to have some of these running parameters accessible in the readme or potentially dictated in error messages.

**Comment by ipeitzsch** [comment]
Yeah, that flag issue was the result of me having the reading comprehension skills of a goldfish. But both of those should be solved in the most recent commit. Thank you for helping test this simulator!
